### PR TITLE
chore: upgrade to the recent nightly

### PR DESCRIPTION
**Pull request by matklad**

### CODE REVIEW

File: engine-tests/src/tests/uniswap.rs [hunk 1]
```diff
@@ -55,13 +55,21 @@ fn test_uniswap_exact_output() {
         context.add_equal_liquidity(LIQUIDITY_AMOUNT.into(), &token_a, &token_b);
     test_utils::assert_gas_bound(profile.all_gas(), 126);
     let wasm_fraction = 100 * profile.wasm_gas() / profile.all_gas();
-    assert!(wasm_fraction >= 23, "{}% not more than 23%", wasm_fraction);
+    assert!(
+        15 <= wasm_fraction && wasm_fraction <= 20,
```
Comment (matklad) [inline engine-tests/src/tests/uniswap.rs:59]: In terms of absolute cost, this went down from `125965273627830` to `116478702274944` :tada:

**Review by sept-en — APPROVED** [review]
Always good to have such updates. Thanks, @matklad!

**Review by birchmd — APPROVED** [review]
Thanks @matklad !

**Review by mrLSD — APPROVED** [review]

**Comment by mrLSD** [comment]
@matklad do we have now a chance to move to `2021 edition`?

**Review by joshuajbouw — APPROVED** [review]
Looks good to me! Thanks man.

**Comment by matklad** [comment]
@mrLSD yeah, though I'd (or someone else :-) ) rather do that in a separate PR

**Comment by matklad** [comment]
edition update in https://github.com/aurora-is-near/aurora-engine/pull/428

**Pull request merged**